graveler/staging_test.go

```diff
@@ -118,6 +116,158 @@ func TestDrop(t *testing.T) {
 	}
 }
 
+func TestDropByPrefix(t *testing.T) {
```

Comment:
can you extract value - context.Background() use the same one for all the func scope